iris_grib/_grib_cf_map.py

```diff
@@ -63,27 +63,36 @@
     G2Param(2, 0, 0, 10): CFName('surface_upward_latent_heat_flux', None, 'W m-2'),
     G2Param(2, 0, 0, 11): CFName('surface_upward_sensible_heat_flux', None, 'W m-2'),
     G2Param(2, 0, 0, 17): CFName('surface_temperature', None, 'K'),
+    G2Param(2, 0, 0, 32): CFName('wet_bulb_potential_temperature', None, 'K'),
     G2Param(2, 0, 1, 0): CFName('specific_humidity', None, 'kg kg-1'),
     G2Param(2, 0, 1, 1): CFName('relative_humidity', None, '%'),
     G2Param(2, 0, 1, 2): CFName('humidity_mixing_ratio', None, 'kg kg-1'),
     G2Param(2, 0, 1, 3): CFName(None, 'precipitable_water', 'kg m-2'),
     G2Param(2, 0, 1, 7): CFName('precipitation_flux', None, 'kg m-2 s-1'),
+    G2Param(2, 0, 1, 9): CFName('stratiform_rainfall_amount', 'Large-scale precipitation (non-convective)', 'kg m-2'),
+    G2Param(2, 0, 1, 10): CFName('convective_rainfall_amount', 'Convective precipitation', 'kg m-2'),
     G2Param(2, 0, 1, 11): CFName('thickness_of_snowfall_amount', None, 'm'),
     G2Param(2, 0, 1, 13): CFName('liquid_water_content_of_surface_snow', None, 'kg m-2'),
+    G2Param(2, 0, 1, 15): CFName('stratiform_snowfall_amount', 'Large-scale snow', 'kg m-2'),
```

Comment:
Julian: your suggestion here was a mapping of `0, 1, 13` to STASH `m01s04i202`. I have instead proposed a mapping that would link `0, 1, 15`.

STASH `m01s04i202` is already mapped to the CF standard name `stratiform_snowfall_amount`, and existing similar mappings for other `stratiform_...` standard names suggest this is the correct course.

Thoughts?

Reply:
Julian has confirmed for me that `0, 1, 15` is the correct mapping.